src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
+tile.openblocks.auto_anvil.name=Automatischer Amboss
+tile.openblocks.auto_anvil.description=Der Automatischer Amboss funktioniert ähnlich wie ein Vanilla-Amboss, außer dass er mit flüssigen Erfahrungspunkten arbeitet und komplett automatisierbar ist.\n\nBenutze die Tabs auf der rechten Seite der Oberfläche, um auszuwählen, welche Seiten des Ambosses für Ein-/Ausgabe von Gegenstände oder Flüssigkeiten vorgesehen sind. Rechts-klicke und ziehe den Amboss im Tab, um ihn zu drehen und andere Seiten auszuwählen!
 
 tile.openblocks.drawing_table.name=Zeichentisch
 tile.openblocks.drawing_table.description=Der Zeichentisch wird gebraucht, um Schablonen herzustellen, die zum Zeichnen gebraucht werden. Gib eine unbearbeitete Schablone in den Zeichentisch und benutze die Pfeile, um die gewünschte Schablone auszuwählen!
 
 tile.openblocks.grave.name=Grab
 tile.openblocks.trophy.name=Trophäe
 tile.openblocks.trophy.entity.name=%s Trophäe
-tile.openblocks.canvasglass.name=Glasleinwand
+tile.openblocks.canvasglass.name=Glas-Leinwand
 tile.openblocks.paint_can.name=Farbeimer
 
 tile.openblocks.sky.normal.name=Himmelsblock
 tile.openblocks.sky.normal.description=Bringe das Außen nach innen mit dem neuen Himmelsblock! Du musst lediglich ein Redstone-Signal geben. Wir bieten auch die invertierte Version an.
 tile.openblocks.sky.inverted.name=Invertierter Himmelsblock
 
 tile.openblocks.radio.name=Radio
-tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Dungeon-Truhen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten.  Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
+tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Truhen aus Verliesen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten. Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
 
-tile.openblocks.xp_shower.name=XP-Dusche
-tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller XP-Saft an und sie wird XP-Orbs ausschütten. Diese Orbs können von einem XP-Gitter oder einer MFR-Kloake aufgesammelt werden.
+tile.openblocks.xp_shower.name=Erfahrungspunkte-Dusche
```

Comment:
Why this change? The former translation was a lot nicer.

Reply:
Not everything has to be translated literally, you know.

Reply:
No :P